api/python/provisioner/commands/check.py

```diff
@@ -137,6 +137,37 @@ def is_set(self) -> bool:
         """
         return self._verdict is not None
 
+    @property
+    def is_critical(self) -> bool:
+        """
+        Return True if check result is marked as critical and False otherwise
+        """
+        return bool(self._critical)
+
+    @property
+    def check_name(self):
+        """
+
+        :return:
+        """
+        return self._check_name
+
+    def set_critical(self) -> None:
```

Comment:
why do we need two separate methods here, can't we just add an argument and use only one?

Reply:
Yes, I agree. Fixed that